README.md

```diff
@@ -253,18 +255,60 @@ export function ProseMirrorEditor() {
 }
 ```
 
-#### `useEditorView`, `EditorViewContext` and `LayoutGroup`
+#### `useEditorEventListener`
+
+`useEditorEventCallback` produces functions that can be passed to React
+components as event handlers. If you need to listen to events that originate
+_within the `contenteditable` node_, however, those event listeners need to be
+registered with the `EditorView`'s `handleDOMEvents` prop.
+
+You can use the `useEditorEventListener` hook to accomplish this. It takes an
+`eventType` and an event listener. The event listener follows the usual
+semantics for ProseMirror's `handleDOMEvents` prop:
+
+- Returning `true` or calling `event.preventDefault` will prevent other
+  listeners from running
+- Returning `true` will not automatically call `event.preventDefault`; if you
+  want to prevent the default contenteditable behavior, you must call
+  `event.preventDefault`.
+
+You can use this hook to implement custom behavior in your NodeViews:
```

Comment:
super minor nit: I think I've been saying "`NodeView`" when talking about the class and "node views" when talking about the concept.

Reply:
Leave it alone, actually. I think I'll take a pass through the whole docs with this in mind after this. I see we've got this in a bunch of places.